src/pages/documentation/media-for-edge-network/tabs/api-reference.md

```diff
@@ -0,0 +1,2117 @@
+                                length: Int, 
+                            streamType: String,
+                             mediaType: MediaType) -> [String: Any]?
+```
+
+**Example**
+
+```swift
+let mediaObject = Media.createMediaObjectWith(name: "video-name",
+                                                id: "videoId", 
+                                            length: 60,
+                                        streamType: MediaConstants.StreamType.VOD, 
+                                         mediaType: MediaType.Video)
+```
+
+#### Objective-C
+
+**Syntax**
+
+```objectivec
++ (NSDictionary<NSString *, id> * _Nullable) createMediaObjectWith:(NSString * _Nonnull) id:(NSString * _Nonnull) length:(NSInteger) streamType:(NSString * _Nonnull) mediaType:(enum AEPEdgeMediaType)
+```
+
+**Example**
+
+```objectivec
+NSDictionary *mediaObject = [AEPMobileEdgeMedia createMediaObjectWith:@"video-name"
+                                                                id:@"video-id" 
+                                                            length:60 
+                                                        streamType:AEPEdgeMediaStreamType.VOD
+                                                         mediaType:AEPEdgeMediaTypeVideo];
+```
+
+<Variant platform="android" api="create-ad-break-object" repeat="8"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static HashMap<String, Object> createAdBreakObject(String name, int position, int startTime);
+```
+
+**Example**
+
+```java
+HashMap<String, Object> adBreakObject = Media.createAdBreakObject("adbreak-name", 1, 0);
+```
+
+#### Kotlin
+
+**Example**
+
+```java
+val adBreakObject = Media.createAdBreakObject("adbreak-name", 1, 0)
+```
+
+<Variant platform="ios" api="create-ad-break-object" repeat="11"/>
+
+Returns a map that contains information about the ad break.
+
+#### Swift
+
+**Syntax**
+
+```swift
+static func createAdBreakObjectWith(name: String,
+                                position: Int, 
+                                startTime: Int) -> [String: Any]?
+```
+
+**Example**
+
+```swift
+let adBreakObject = Media.createAdBreakObjectWith(name: "adbreak-name", 
+                                              position: 1, 
+                                             startTime: 0)
+```
+
+#### Objective-C
+
+**Syntax**
+
+```objectivec
++ (NSDictionary  <NSString *, id> * _Nullable) createAdBreakObjectWith:(NSString * _Nonnull)position:(NSInteger) startTime:(NSInteger)
+```
+
+**Example**
+
+```objectivec
+NSDictionary *adBreakObject = [AEPMobileEdgeMedia createAdBreakObjectWith:@"adbreak-name" 
+                                                             position:1 
+                                                            startTime:0];
+```
+
+<Variant platform="android" api="create-ad-object" repeat="8"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static HashMap<String, Object> createAdObject(String name, String adId, int position, int length);
```

Comment:
Use `id` instead of `adId`. We can update the Android code with the same to match the iOS implementation.